File: resourcespoet/src/test/java/com/commit451/resourcespoet/DimensionTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

/**
 * Tests the resources creation
 */
public class DimensionTest {

    @Test
    public void dimensionTest() throws Exception {
        ResourcesPoet poet = ResourcesPoet.create()
                .addDimension("margin", "2dp");

        TestUtil.assertEquals("dimension.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/DimensionTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class DimensionTest {

    @Test
    fun dimensionTest() {
        val poet = ResourcesPoet.create()
            .addDimension("margin", "2dp")

        TestUtil.assertEquals("dimension.xml", poet)
    }
}
